src/__tests__/surveys.test.ts

```diff
@@ -52,6 +52,35 @@ describe('surveys', () => {
             questions: [{ type: SurveyQuestionType.Open, question: 'what is a moblin?' }],
         } as unknown as Survey,
     ]
+    const surveysWithEvents: Survey[] = [
+        {
+            name: 'first survey',
+            id: 'first-survey',
+            description: 'first survey description',
+            type: SurveyType.Popover,
+            questions: [{ type: SurveyQuestionType.Open, question: 'what is a bokoblin?' }],
+            conditions: {
+                events: ['user_subscribed', 'user_unsubscribed', 'billing_changed', 'billing_removed'],
```

Comment:
can we make this like so:

```suggestion
                events: [{name: 'user_subscribed'}, ...],
```

so that when it comes time to add other metadata for targeting, it's much easier to extend?

Actually, perhaps go even a step further, like we do with group property filters (see https://github.com/PostHog/posthog/blob/master/posthog/models/property/property.py#L330 ) and make this like ->

```
events: {
    "type": "AND" // or "OR"
    "values": [{"name": "X"}, {"name": "Y"}]
}
```

Unless you think this is too premature. I can already see us extending to these use cases very soon though.

Reply:
We can skip the conditional logic now and go with 

``` json
"events": {
    "values": [{"name": "X"}, {"name": "Y"}]
}
```